dist/flight/flight.controller.d.ts

```diff
@@ -0,0 +1,20 @@
+import { FlightService } from "./flight.service";
+export declare class FlightController {
+    private readonly flightService;
+    constructor(flightService: FlightService);
+    bookFlight(flightFrom: string, flightTo: string, flightDate: string, flightPrice: number, flightSeats: number, flightCompany: string): Promise<{
+        id: number;
+    }>;
+    cancelFlight(id: string): Promise<any>;
+    getBookingList(): Promise<{
```

Comment:
try to create separate return type object and can be reused across different layers like service/controller

Reply:
I need some help with this issue.